ts/features/payments/bizEventsTransaction/components/PaymentsBizEventsListItemTransaction.tsx

```diff
@@ -0,0 +1,89 @@
+import { Avatar, ListItemTransaction } from "@pagopa/io-app-design-system";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
+import React from "react";
+import { getAccessibleAmountText } from "../../../../utils/accessibility";
+import { format } from "../../../../utils/dates";
+import { TransactionListItem } from "../../../../../definitions/pagopa/biz-events/TransactionListItem";
+import { getTransactionLogo } from "../../common/utils";
+import { formatAmountText } from "../utils";
+import I18n from "../../../../i18n";
+
+type Props = {
+  transaction: TransactionListItem;
+  onPress?: () => void;
+};
+
+const PaymentsBizEventsListItemTransaction = ({
+  transaction,
+  onPress
+}: Props) => {
+  const recipient = pipe(
+    transaction.payeeName,
+    O.fromNullable,
+    O.getOrElse(() => "")
+  );
```

Comment:
```suggestion
  const recipient = transaction.payeeName || "";
```

I know fp-ts is cool, in this case it's a bit excessive. What do you think?

Reply:
You are absolutely right! But I agree more on it being excessive, less on fp-ts being cool 😆

Addressed into 90bdac3da7e0ddc57949cbdd54da09aa5dc0d231